components/forms/conditionOfJoining/CojView.tsx

```diff
@@ -107,7 +101,7 @@ function CojDeclarationSection({
                 </SummaryList.Value>
               </SummaryList.Row>
             </SummaryList>
-            {!canEdit ? (
+            {!!signedDate ? (
```

Comment:
I think you can rely on truthy/falsy for this ( one `!` ) otherwise you'll get one of those **Redundant double negation.** code smells.